### DISCUSSION TITLE
https://plugin-comment2.vuejs.press/demo.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#749] [Announcements]
# https://plugin-comment2.vuejs.press/demo.html

We provide a demo using vuepress-plugin-comment2 and @vuepress/theme-default for you to better understand how to use the plugin. Source Code (https://github.com/vuepress-theme-h...

https://plugin-comment2.vuejs.press/demo.html

**Comment by JGMa-java** [comment]
youxi dadadi
